src/include/sof/audio/ipc-config.h

```diff
@@ -20,6 +20,8 @@ struct ipc_config_host {
 	uint32_t no_irq;		/**< don't send periodic IRQ to host/DSP */
 	uint32_t dmac_config;		/**< DMA engine specific */
 	uint32_t dma_buffer_size;	/**< Requested DMA buffer size */
+	uint32_t feature_mask;		/**< copier feature mask (set directly from */
+					/**< ipc4_copier_module_cfg on init)*/
```

Comment:
is this a correct doxygen format? Shouldn't the description of `feature_mask` be in a single comment block?